shell/common/gin_helper/event_emitter_caller.h

```diff
@@ -82,7 +83,7 @@ template <typename T, typename... Args>
 v8::Local<v8::Value> CallMethod(gin::Wrappable<T>* object,
                                 const char* method_name,
                                 Args&&... args) {
-  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+  v8::Isolate* isolate = electron::JavascriptEnvironment::GetIsolate();
```

Comment:
A file in `shell/common` shouldn't depend on `shell/browser`, this won't work in a renderer context

Reply:
you're right - only `shell/browser` file changes left now!